cobc/typeck.c

```diff
@@ -3814,8 +3745,21 @@ get_value (cb_tree x)
 	}
 }
 
+void
+load_collating_tables (void)
+{
+	static int coltab_loaded = 0;
+	if (coltab_loaded) {
+		return;
+	}
+	if (cob_load_collation (cb_ebcdic_table, ebcdic_to_ascii, ascii_to_ebcdic) < 0) {
+		cobc_err_exit (_("invalid parameter: %s"), "-febcdic-table");
```

Comment:
Alright, that would mean we'd always load the collating table, instead of loading it only if needed (i.e. when a non-native alphabet is used somewhere in the program). Is that alright ?

Reply:
I don't see a way for validating the option otherwise, do you? As an alternative this may be split into "validation" and actual load (we can assume that the file doesn't change in between), but that's possibly not worth it.

Doing the "load" part for anything "default" as needed should stay in, of course.